.github/workflows/cache-factory.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       - uses: actions/checkout@v3
 
-      - uses: actions-rs/toolchain@16499b5e05bf2e26879000db0c1d13f7e13fa3af # v1.0.7
+      - uses: dtolnay/rust-toolchain@v1
```

Comment:
Thanks!

We have the policy of pinning third-party actions to a hash. See https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions.

Please use the hash and write the comment next to it. Dependabot can parse that and will bump it for is when there is a new release.

Reply:
I see, updated the PR